pytgcalls/pytgcalls/mtproto/telethon_bridge.py

```diff
@@ -25,14 +25,14 @@
     GroupcallSsrcDuplicateMuchError as TelethonGroupcallSsrcDuplicateMuchError,
 )
 from telethon.events import Raw, StopPropagation
-from telethon.tl import functions
+from telethon.tl import functions, TLObject
```

Comment:
where are you using this import?

Reply:
I used it before to check if an object passed to `get_and_set_group_call` is an TLObject but I turned it down.It's not needed